### PR TITLE
repositioning ui on window resize

### LINKED ISSUE #552: Reposition nubs on window resize
Right now the nub position will shift when a browser window is resized and look visually broken.

### PR DESCRIPTION
**Pull request by gvn**
Closes #552

### CODE REVIEW

File: src/pages/project/cartzoom.js [hunk 1]
```diff
@@ -131,6 +130,22 @@ module.exports = {
       btnZoomOut.style.top = (boundingBox.top - (btnZoomOut.clientHeight / 2)) + 'px';
       btnZoomOut.style.left = (boundingBox.left - (btnZoomOut.clientWidth / 2)) + 'px';
     }
+  },
+
+  componentDidMount: function () {
```
Comment (k88hudson) [inline src/pages/project/cartzoom.js:135]: you may as well add a `removeEventListener` for good measure on `componentDidUnmount`, right?

**Comment by k88hudson** [comment]
Added a small suggestion, works well though

**Comment by gvn** [comment]
When it's unmounted hasn't the webview also been destroyed?

**Comment by k88hudson** [comment]
Yeah... I guess in general it just worries me if we end up repurposing the component or something. But I guess it's fine

**Pull request merged**